ansible/site.yml

```diff
@@ -0,0 +1,22 @@
+- hosts: bootstrap
+  remote_user: ubuntu
+  become: yes
+  become_method: sudo
+  roles:
+    - bootstrap
```

Comment:
Yep, the initial node has also bootkube provisioned. The bootstrap node provides a temporary API-Server (by running "bootkube start") which the kubelets can connect to to start the control-plane components (apiserver, cm, scheduler, etcd). After that successfully happened, bootkube will quit itself. This is a process which happens only once.

Then each node is exactly the same, there's just a kubelet running. The only difference between master and worker nodes is the `--node-labels=master=true` parameter for the kubelet.